mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Kitto.Mixfile do
     [{:cowboy, "~> 1.0.0"},
      {:plug, "~> 1.3"},
      {:poison, "~> 3.0"},
-     {:fs, "~> 2.11.0"},
+     {:fs, github: "synrc/fs"},
```

Comment:
I did. It doesn't work. Needs the new `fs` updates on the `master` branch to be released.